tests/metrics/gha-run.sh

```diff
@@ -91,12 +91,22 @@ function run_test_launchtimes() {
 	bash tests/metrics/time/launch_times.sh -i public.ecr.aws/ubuntu/ubuntu:latest -n 20
 }
 
+function run_test_blogbench() {
+	hypervisor="${1}"
+	info "Running Blogbench test using ${hypervisor} hypervisor"
+
+	create_symbolic_links "${hypervisor}"
+	bash tests/metrics/storage/blogbench.sh
+}
+
 function main() {
 	action="${1:-}"
 	case "${action}" in
 		install-kata) install_kata ;;
 		run-test-launchtimes-qemu) run_test_launchtimes "qemu" ;;
 		run-test-launchtimes-clh) run_test_launchtimes "clh" ;;
+		run-test-blogbench-qemu) run_test_blogbench "qemu" ;;
+		run-test-blogbench-clh) run_test_blogbench "clh" ;;
```

Comment:
I'm missing something here, but shouldn't we also add those to the netrics yaml action?

I'd take the following path:
* Add to the yaml
* Add it here, with short-circuit to always return true

In a separe PR:
* Actually enable the tests in the script and make sure they're passing.

Reply:
@dborquez You will need to modify `run-k8s-tests-on-aks.yaml` in order to invoke these tests.

Reply:
@fidencio I've make the function that runs the test in order to returns zero
I've added the test to the workflow, but I will wait that the memory test to land, and rebase this PR w/ the 'matrix' usage  update.

Reply:
@amshinde I took the matrix idea from there, thank you :)